api/src/controllers/user.controller.ts

```diff
@@ -293,6 +294,48 @@ export const claimPitch = async (
   );
 };
 
+// Returns all pitches that a user is a member of
+export const pitches = async (req: Request, res: Response): Promise<void> => {
+  const populateType = extractPopulateQuery(req.query);
+  const options = extractOptions(req.query);
+
+  const pitches = await PitchService.getMemberPitches(req.params.id, options);
+
+  if (pitches === undefined) {
+    sendNotFound(res, `User not found with id ${req.params.id}`);
+    return;
+  }
+
+  sendSuccess(res, 'Successfully retrieved all user pitches', {
+    data: await populatePitches(pitches.data, populateType),
+    count: pitches.count,
+  });
+};
+
+export const submittedClaims = async (
```

Comment:
In the client side code, using an endpoint is simpler for the implementation since it works like the pitchdoc where, depending on the tab, a different URL is used to fetch data. I've decided to delete this endpoint though since it can be replaced with a query string on the other endpoint I created.